.circleci/src/jobs/check_administration.yml

```diff
@@ -0,0 +1,31 @@
+docker:
+  - image: cimg/node:19.1.0
+environment:
+working_directory: ~/project/administration
+steps:
+  - checkout:
+      path: ~/project
+  - restore_cache:
+      keys:
+        - v2-node-modules-{{ checksum "package.json" }}-{{checksum "package-lock.json" }}
```

Comment:
I think it would be nice, if the cache key also includes  `administration`-string, so we know this keys belong to the node_modules in the adminsitration directory.